src/resources/views/crud/buttons/on_the_fly/loader.blade.php

```diff
@@ -0,0 +1,138 @@
+@push('loading_styles')
```

Comment:
I added this loader because if you are loading a really huge crud (i tried with monster - creating a monster on the fly) it must call alot of JS and CSS for the fields and it delays the modal for about 2/3 seconds ... that's why this loader. 

This problem also relates with the fact of us loading 4/5/ ... times the select2.js, select2.css etc etc

Reply:
Also, I don't think its place is inside the ```crud/buttons``` folder. It's something that only the on-the-fly fields use. So maybe a better place for it would be inside ```crud/fields/inc```?